front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/tag/EntityTags.groovy

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.netflix.spinnaker.front50.model.tag
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter
+import com.fasterxml.jackson.annotation.JsonAnySetter
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.netflix.spinnaker.front50.model.Timestamped
+
+class EntityTags implements Timestamped {
+  String id
+  String idPattern   // pattern used to construct the `id`, ie. {{entityType}}__{{entityId}}__{{account}}__{{region}}
+
+  Long lastModified
+  String lastModifiedBy
+
+  Map<String, Object> tags = [:]
+  EntityRef entityRef
+
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  static class EntityRef {
+    private Map<String, Object> attributes = new HashMap<String, Object>()
+
+    String entityType
+    String entityId
+
+    @JsonAnyGetter
+     Map<String,Object> attributes() {
```

Comment:
Before the attributes were just strings. Now it looks like they can be composite. Does that pose a problem querying them back? Just asking. I didnt look up what JsonAnyGetter is.